masterlist.yaml

```diff
@@ -7084,12 +7099,43 @@ plugins:
       # version: 1.58
       - crc: 0xD7423A4E
         util: 'SSEEdit v4.0.0'
+  - name: 'imp_helm_Immersive_Armors.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/12405/'
+        name: 'Improved Closedfaced Helmets Patches'
+    req:
+      - 'imp_helm_legend.esp'
+    after:
+      - 'Hothtrooper44_ArmorCompilation.esp'
```

Comment:
Not required, `Hothtrooper44_ArmorCompilation.esp` is a master of this patch.

![image](https://user-images.githubusercontent.com/1944639/57931323-1bf30780-78b0-11e9-9d20-014235b34c7f.png)